src/algorithms/optimization/peps_optimization.jl

```diff
@@ -74,6 +74,8 @@ function _OptimizationAlgorithm(;
     tol=Defaults.optimizer_tol,
     maxiter=Defaults.optimizer_maxiter,
     verbosity=Defaults.optimizer_verbosity,
+    ls_maxiter=Defaults.ls_maxiter,
+    ls_maxfg=Defaults.ls_maxfg,
     lbfgs_memory=Defaults.lbfgs_memory,
     # TODO: add linesearch, ... to kwargs and defaults?
```

Comment:
Is this TODO now obsolete?

Reply:
Not really, there's still no way to really specify the other line search parameters (e.g. those that affect step acceptance) or pass in an actual line search algorithm instance. So I would leave this, but tackle it later when we actually get to solving our line search issues. For now, we can still pass in a concrete optimization algorithm where we set the line search algorithm parameters beforehand, so it would just be an extra convenience thing.